src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -98,7 +98,9 @@ public function denormalize($data, $class, $format = null, array $context = [])
     {
         // Avoid issues with proxies if we populated the object
         if (isset($data['@id']) && !isset($context[self::OBJECT_TO_POPULATE])) {
-            if (isset($context['api_allow_update']) && true !== $context['api_allow_update']) {
+            if (isset($context['api_allow_update']) && true !== $context['api_allow_update']
```

Comment:
WDYT about `if (!($context['api_allow_update'] ?? false) && !($context['api_resource_is_relation'] ?? false))` ?

Reply:
I think you mean `if (!($context['api_allow_update'] ?? true)  ...` because now true is the default value for `api_allow_update` (another way of saying is that an unset api_allow_update wont go if the `if`).
The only difference here is if the value is set but not boolean, but that "should" not append (except if a user set it manually in the serialization context).

I like it but can maybe be hard to read, but why not.

As a matter of fact, `api_allow_update` is always set in the `SerializerContextBuilder` so I'm wondering if someone can denormalize an item without going through the context builder ? 
If no, the code can be `true !== $context['api_allow_update']` (but it may be safer to keep the `isset` check.